### ISSUE TITLE
What is the status on this project ?

### ISSUE THREAD

**Issue by HHK1** [Dev1an/MsgPack#1]
Hi !
Thanks for sharing your implementation of messagePack with the new swift4 Codable protocols, I was looking for exactly this 👍 

Is it something I could use in a production environment ? I can see you are referenced on the message pack website, but I don't know if you plan on spending time maintaining the project. 

Cheers

**Comment by Dev1an** [comment]
Well, I started this project as an experiment and I am not using it in any production app. Swift 4 is still in beta and the documentation on how to implement custom encoders and decoders is very limited.

I am not actively working on this right now but if you find bugs, file them and I will resolve them in my spare time...

**Comment by B00jan** [comment]
I'm new at programming, so sorry if the question is stupid. Is there a way to pack a whole class, and not do every single property separately? For example:
class Person{

var name: String
var lastName: String
var age: Int
}
let me = Person(name:"First", lastName: "Last", age: 2332)

let packed = me.pack
...
var unpacked = receivedData.unpack

or something like this...

**Comment by Dev1an** [comment]
Yes, see the second part (_"Encode custom types with Encodable"_) of the example in the readme. You should make your class conform to `Codable`

**Comment by B00jan** [comment]
Thank you for your fast response!

**Comment by kirilltitov** [comment]
What about those "not implemented" parts? Are you going to implement them?

**Comment by Dev1an** [comment]
@kirilltitov Since Flight-School has implemented the entire protocol already, I think it may be better to use his library. https://github.com/Flight-School/MessagePack
